hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -631,6 +631,13 @@ public static void assertIterableEquals(Iterable iterable, Object... values) {
         assertEquals(expected, actual);
     }
 
+    public static void assertSetEquals(Set actual, Object... values) {
```

Comment:
Good point. 

We need to clean up the testing because the Asserts are in multiple classes.